README.md

```diff
@@ -241,6 +242,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [LimeChat](http://limechat.net/mac/)
   - [LittleSnitch](http://www.obdev.at/products/littlesnitch/)
   - [Livestreamer](http://livestreamer.tanuki.se/)
+  - [LSColors](https://github.com/TroyWilson1)
```

Comment:
The url should be https://github.com/trapd00r/LS_COLORS

Sorry, I'm still learning how this works.